### PR TITLE
Improve documentation when building with podman on non-SLE host

### PR DESCRIPTION
**Pull request by fcrozat**
Add alternative documentation for building with podman on non-SLES host

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +252,17 @@ Please keep in mind that it is not possible to use `container-suseconnect` or
 `zypper` within the container after the build, because the secrets are not
 available any more.
 
+
+### Alternative when using podman as container runtime
+
+When using `podman` as container runtime, you can add the following lines to `/etc/containers/mounts.conf` (when building as `root`) or `~/.config/containers/mounts.conf` (when building as regular user):
```
Comment (dmpop) [inline README.md:258]:
```suggestion
When using `podman` as container runtime, add the following to `/etc/containers/mounts.conf` (when building containers as `root`) or `~/.config/containers/mounts.conf` (when building containers as a regular user):
```

File: README.md [hunk 2]
```diff
@@ -252,6 +252,17 @@ Please keep in mind that it is not possible to use `container-suseconnect` or
...
+```
+<path_on_host>/SUSEConnect:/etc/SUSEConnect
+<path_on_host>/SCCcredentials:/etc/zypp/credentials.d/SCCcredentials
+```
+
+No further change are needed in `Dockerfile`. This change is already part of `podman` package on SLES host.
```
Comment (dmpop) [inline README.md:264]:
```suggestion
No further change are needed in `Dockerfile`.
```

**Comment by lagartoflojo** [comment]
Title: "building with podman on SLE host"
Summary: "building with podman on **non-SLES** host"

I think the description needs to be corrected =)

**Comment by fcrozat** [comment]
> Title: "building with podman on SLE host" Summary: "building with podman on **non-SLES** host"
> 
> I think the description needs to be corrected =)

fixed ;)

**Comment by lagartoflojo** [comment]
But I think you meant to say "building with podman on SLES host," no? AFAIU the description was wrong, not the title.
Because non-SLES hosts don't have the SUSEConnect config nor the zypper credential files.

**Comment by fcrozat** [comment]
> But I think you meant to say "building with podman on SLES host," no? AFAIU the description was wrong, not the title. Because non-SLES hosts don't have the SUSEConnect config nor the zypper credential files.

No, the podman changes are already part of SLES podman. The changes descripted in this PR are needed on non-SLES host, as an addition to the section "Building images on non SLES distribution" which is docker focused.

**Comment by lagartoflojo** [comment]
Ahh got it, thanks. You might also need to update the commit message then =)

**Comment by fcrozat** [comment]
> Ahh got it, thanks. You might also need to update the commit message then =)

Oh my... Fixed ;)

File: README.md [hunk 3]
```diff
@@ -252,6 +252,17 @@ Please keep in mind that it is not possible to use `container-suseconnect` or
 `zypper` within the container after the build, because the secrets are not
 available any more.
 
+
+### Alternative when using podman as container runtime
+
+When using `podman` as container runtime, add the following to `/etc/containers/mounts.conf` (when building containers as `root`) or `~/.config/containers/mounts.conf` (when building containers as a regular user):
```
Comment (dcermak) [inline README.md:258]:
```suggestion
When using `podman` as the container runtime, add the following to `/etc/containers/mounts.conf` (when building containers as `root`) or `~/.config/containers/mounts.conf` (when building containers as a regular user):
```

**Review by pdostal — APPROVED** [review]
This is nice addition

**Pull request merged**